### PR TITLE
RATIS-2067 Enhance content for concept in ratis-docs

### PR DESCRIPTION
**Pull request by OneSizeFitsQuorum**
see [jira](https://issues.apache.org/jira/browse/RATIS-2067)

### CODE REVIEW

**Comment by OneSizeFitsQuorum** [comment]
@szetszwo PTAL

File: ratis-docs/src/site/markdown/snapshot.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ Note that Ratis imposes a minimal creation gap between two subsequent snapshot c
   ```java
   // SnapshotManagementApi
   RaftClientReply create(long timeoutMs) throws IOException;
+  RaftClientReply create(boolean force, long timeoutMs) throws IOException {
```
Comment (szetszwo) [inline ratis-docs/src/site/markdown/snapshot.md:78]: Use `;` at the end.

**Review by szetszwo — COMMENTED** [review]
@OneSizeFitsQuorum , thanks for working on this!  Let's use short lines, which will be easier to make changes later on.  I have some suggestions; see https://issues.apache.org/jira/secure/attachment/13068572/1073_review.patch

File: ratis-docs/src/site/markdown/snapshot.md (hunk 1 above)
Comment (OneSizeFitsQuorum) [reply ratis-docs/src/site/markdown/snapshot.md:78]: Fixed

**Comment by OneSizeFitsQuorum** [comment]
@szetszwo Thanks for your detailed review~

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Pull request merged**